### PR TITLE
Wip solution seed data

### PR DESCRIPTION
**Pull request by vinnyalfieri**
@jmburges @deniznida @ipc103 

I expanded the solution to show the actual Database data to display on the screen for each view. Also added a "Figures" view to display more data from the Database. 

This introduces passing data from your controller to the actual view via class_variables

### CODE REVIEW

File: app/views/figures/index.erb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<table>
+  <th>Person Name</th>
+  <tr>
+    <% @figure.each do |figures| %>
```
Comment (jmburges) [inline app/views/figures/index.erb:4]: @figure is your variable name and `figures` is the block variable? I you sure it shouldn't be `@figures` is the instance variable and `figure` is the block variable?

File: app/views/landmarks/index.erb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<table>
+  <th>Name</th>
+  <th>Year Completed</th>
+  <tr>
+    <% @landmark.each do |land| %>
```
Comment (jmburges) [inline app/views/landmarks/index.erb]: shouldn't that be the `@landmarks`? It's multiple landmarks right?

Comment (jmburges) [inline app/views/landmarks/index.erb]: also `land` as a block variable doesn't make sense. It's not all of a sudden a piece of land right? isn't is `landmark`?

**Comment by jmburges** [comment]
back at you

**Comment by vinnyalfieri** [comment]
Updated the seed data to remove unused elements.
Updated the Landmarks controller to use proper naming convention. Pluralized @landmark since it is supposed to hold MANY landmarks.

@jmburges @AnnJohn closes issue #6

**Pull request merged**